modules/configuring-microshift-virtual-machine.adoc

```diff
@@ -0,0 +1,53 @@
+// 
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-microshift-virtual-machine"]
+= Configuring the MicroShift virtual machine
+:context: creating-microshift-development-environment
+
+Use the following procedure to configure a virtual machine for MicroShift 
+
+.Prerequisites 
+
+* You have logged into your development virtual machine using your MicroShift user credentials. 
+
+.Procedure 
+
+. Run the following command to configure root privileges: 
++
+[source,terminal]
+----
+$ echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
+----
+
+. Run the following command to upgrade your system:
++
+[source,terminal]
+----
+$ sudo dnf update -y
+----
+
+. Run the following command to install basic dependencies: 
++
+[source,terminal]
+----
+$ sudo dnf install -y git cockpit make gcc selinux-policy-devel rpm-build bash-completion
+----
+
+. Run the following command to enable the remote cockpit console:
++
+[source,terminal]
+----
+$ export GO_VER=1.18.7; curl -L -o go${GO_VER}.linux-amd64.tar.gz https://go.dev/dl/go${GO_VER}.linux-amd64.tar.gz &&
+    sudo rm -rf /usr/local/go${GO_VER} && \
+    sudo mkdir -p /usr/local/go${GO_VER} && \
+    sudo tar -C /usr/local/go${GO_VER} -xzf go${GO_VER}.linux-amd64.tar.gz --strip-components 1 && \
+    sudo rm -rfv /usr/local/bin/{go,gofmt}
+    sudo ln --symbolic /usr/local/go1.18.7/bin/go /usr/local/go1.18.7/bin/gofmt /usr/local/bin/ && \
+    rm -rfv go${GO_VER}.linux-amd64.tar.gz
+----
+
```

Comment:
```suggestion
```

The commands above install golang (not cockpit). Users won't need to do that step, so you could cut lines 40-52.